### ISSUE TITLE
Frontend- page to edit personal information of a digital identity

### ISSUE THREAD

**Issue by TamaraHoock** [amosproj/amos2022ss04-digital-identity#11]
## User story
1. As a user (HR employee)
2. I want / need to edit personal information of a registered employee
3. So that I can update an employee's personal information

## Acceptance criteria
* should be accessable via "edit"-button in the overview-tab 
* all information should be changeable
* should trigger change-process in the backend

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
